### ISSUE TITLE
[PSC-001] How to handle McDonald's Collection 2018/2019

### ISSUE THREAD

**Issue by immewnity** [Pokemon-Standards-Consortium/standards#1]
France has a different set than the rest of the territories for these two McDonald's promotions, with the 2019 sets confusingly sharing the same set symbol.

How do we distinguish these expansions? Something like "MCD19FR" and "MCD19INTL"? Do we adapt this to earlier McDonald's sets that don't have a difference, for consistency's sake?

**Comment by immewnity** [comment]
Not a draft yet

**Comment by immewnity** [comment]
Reopening now that PSC 001 is actually a draft.

**Comment by Dillonzer** [comment]
I agree with MCD{YEAR}{LOCALIZATION}. Do we ever have more than one a year?

**Comment by MatiasOETamminen** [comment]
Not yet, but might in the future.

**Comment by immewnity** [comment]
> 
> 
> I agree with MCD{YEAR}{LOCALIZATION}. Do we ever have more than one a year?

...kinda. So, the cards that came to the US in 2019 came to the UK in 2020.

**Comment by MatiasOETamminen** [comment]
> > I agree with MCD{YEAR}{LOCALIZATION}. Do we ever have more than one a year?
> 
> ...kinda. So, the cards that came to the US in 2019 came to the UK in 2020.

But there is no MCD2020

**Comment by immewnity** [comment]
> But there is no MCD2020

Right - in this scenario, it doesn't lead to confusion, but the possibility is there (and I haven't checked to see if the same thing happened in previous years)

**Comment by immewnity** [comment]
So, I think we're pretty well agreed on naming French sets like `MCD19FR`. What about the non-French ones? `MCD19INTL` or simply `MCD19`?

**Comment by MatiasOETamminen** [comment]
I would prefer "MCD19" for the English one and "MCD19F" for the French one to maximize overlap with existing abbreviations. I would, however, call MCD13 "MCD13" even though it's in French, because it's only in French.

**Comment by DillonBast-iQ** [comment]
> call MCD13 "MCD13" even though it's in French, because it's only in French.

I disagree with this. If it's French it should still be MCD13F and MCD13 just won't exist. 

> What about the non-French ones? `MCD19INTL` or simply `MCD19`?
I think non-French (if they are English) should be simply MCD19. This leaves the options for other languages to get sets and easily append the language to the abbreviations.

**Comment by Dillonzer** [comment]
> call MCD13 "MCD13" even though it's in French, because it's only in French.

I disagree with this. If it's French it should still be MCD13F and MCD13 just won't exist.

> What about the non-French ones? MCD19INTL or simply MCD19?

I think non-French (if they are English) should be simply MCD19. This leaves the options for other languages to get sets and easily append the language to the abbreviations.

**Comment by immewnity** [comment]
`MCD13F`/`MCD13FR` makes more sense to me, IMO - if we consider them in a tabular format, it's more consistent to use the region code:

| International | France-specific |
|---------------|-----------------|
| MCD11         |                 |
| MCD12         |                 |
|               | MCD13F(R)         |
| MCD14         |                 |
| MCD15         |                 |
| MCD16         |                 |
| MCD17         |                 |
| MCD18         | MCD18F(R)         |
| MCD19         | MCD19F(R)         |
| MCD21         |                 |

On a related note... `MCD##F` or `MCD##FR`? Personally, I'm partial to the latter in order to match ISO 3166-1 alpha-2, as well as for easier clarity for someone who doesn't know the situation, but understand using `F` since it's shorter and already in use for Cardmarket.

**Comment by egonolieux** [comment]
I prefer `FR` over `F` as well for the reasons you mentioned. Also, keep in mind that these sets (including the English one) are part of the international TCG, so using the `INTL` suffix seems inappropriate here.

There is an argument to use the `EN` suffix, but I feel somewhat conflicted about this, as English is the de-facto reference for the international TCG.

**Comment by immewnity** [comment]
`EN` would definitely be a misnomer, since the promotion has been done in other languages using the same set structure.

**Comment by egonolieux** [comment]
Hmm, true, only using the regional suffix for region-specific anomalies seems like the best option then.
